### PR TITLE
Adding the registration and first sponsor.

### PR DESCRIPTION
**Pull request by cobusbernard**
Ready for merging.

### CODE REVIEW

File: data/sponsors/2016-offerzen.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+name: "OfferZen"
+url: "https://www.offerzen.com/?utm_source=devopsdays&utm_medium=sponsor&utm_campaign=1"
```
Comment (kmugrage) [inline data/sponsors/2016-offerzen.yml]: Is this UTM good for any DevOpsDays event or should the yml be namespaced to be Cape Town specific?

**Comment by kmugrage** [comment]
Added a question about the UTM, wondering if it's Cape Town specific. If so, perhaps the yml file should be namespaced to reflect that? (eg, 2016-sponsor-capetown)

**Comment by kmugrage** [comment]
Can you please resize the image to 200px square?

**Comment by cobusbernard** [comment]
Will change to be Cape Town specific and check the image, I requested a 200px square, assumed :)

**Pull request merged**